RealmSwift/List.swift

```diff
@@ -69,6 +70,10 @@ public final class List<T: Object>: ListBase {
 
     // MARK: Initializers
 
+    fileprivate override init!(array: RLMArray<RLMObject>!) {
```

Comment:
I guess the IUO is only due to some of our internal headers lacking nullability annotations?